extensions/settings/pkg/config/config.go

```diff
@@ -35,15 +35,15 @@ type Config struct {
 
 // Asset defines the available asset configuration.
 type Asset struct {
-	Path string `yaml:"path" env:"SETTINGS_ASSET_PATH"`
+	Path string `yaml:"path" env:"SETTINGS_ASSET_PATH" desc:"Serve settings assets from a path on the filesystem instead of the builtin assets."`
```

Comment:
What does that mean and why is this necessary?
Reading this sentence many times, I find this hard to understand.